sdl2promela/translator.py

```diff
@@ -132,7 +132,16 @@ def __generate_variable_name(
     constant_reference: sdlmodel.ConstantReference,
     toplevel: bool,
 ):
-    return VariableReferenceBuilder(constant_reference.constantName.lower()).build()
+    if constant_reference.constantName == "{}_ctxt".format(
```

Comment:
I think it would be more readable to have a function named something like: isConstantSourcedFromFunctionType... This is just a quick proposal, there might be a shorter, yet still decriptive name

Reply:
And the comment could be moved to that function. Alternativelly, the entire constant name processing could be factored into a small function, making this one a one-liner to fit into the dispatcher.

Reply:
done